blockchain/chain/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
-
+#![feature(const_option)]
```

Comment:
We want to be stable so this is not an option.

Reply:
You can manually check for `None`, though.

Reply:
Removed it.